edc-extensions/tokenrefresh-handler/src/main/java/org/eclipse/tractusx/edc/common/tokenrefresh/TokenRefreshHandlerImpl.java

```diff
@@ -178,6 +178,7 @@ private Result<Request> createTokenRefreshRequest(String refreshEndpoint, String
 
         return success(new Request.Builder()
                 .addHeader("Authorization", bearerToken)
+                .addHeader("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Why does it need to be a `application/x-www-form-urlencoded`? And why does the body need to be a `new byte[0]`? Why can't it be empty?

Reply:
Okay, found the answer for the first question: OAuth2 standard

Reply:
the header is needed because of the [spec](https://datatracker.ietf.org/doc/html/rfc6749#section-3.1)
the body is there because post request must have a body, at least if we're using OkHttp